### PR TITLE
Fix indentation of inserted use statements

### PR DESCRIPTION
**Pull request by flodiebold**

### CODE REVIEW

**Comment by flodiebold** [comment]
Hmm actually maybe there's a better logic for this :thinking:

**Comment by flodiebold** [comment]
This might be slightly more correct now.

File: crates/assists/src/utils/insert_use.rs [hunk 1]
```diff
@@ -133,6 +142,13 @@ pub(crate) fn insert_use(
             _ => (),
         }
 
+        // only add indentation *after* our stuff if there's another node directly after it
+        if add_blank.has_after() && matches!(insert_position, InsertPosition::Before(_)) {
+            if let Some(indent) = indent {
+                buf.push(indent);
+            }
+        }
```
Comment (matklad) [inline crates/assists/src/utils/insert_use.rs:150]: Would a `mark::hit` be useful here?

Comment (flodiebold) [reply crates/assists/src/utils/insert_use.rs:155]: Sure, I added some marks.

**Comment by flodiebold** [comment]
:thinking: 

bors r+

**Comment by flodiebold** [comment]
Hrm. Sometimes these mark checks don't seem to fail for me locally.

**Pull request merged**